syslog-ng/Dockerfile

```diff
@@ -4,12 +4,11 @@ MAINTAINER Andras Mitzki <andras.mitzki@balabit.com>
 RUN apt-get update -qq && apt-get install -y \
     wget
 
-RUN wget -qO - http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng-3.6/Debian_8.0/Release.key | apt-key add -
-RUN echo 'deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng-3.6/Debian_8.0 ./' | tee --append /etc/apt/sources.list.d/syslog-ng-obs.list
+RUN wget -qO - http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/Debian_8.0/Release.key | apt-key add -
+RUN echo 'deb http://download.opensuse.org/repositories/home:/laszlo_budai:/syslog-ng/Debian_8.0 ./' | tee --append /etc/apt/sources.list.d/syslog-ng-obs.list
 
 RUN apt-get update -qq && apt-get install -y \
-    syslog-ng \
-    syslog-ng-mod*
+    syslog-ng
```

Comment:
In issue https://github.com/balabit/syslog-ng/issues/677 we verified that "syslog-ng" package installs all module packages for syslog-ng, so we do not need to install "syslog-ng-mod*".